system-test/spanner.ts

```diff
@@ -64,6 +64,10 @@ describe('Spanner', () => {
     },
   };
 
+  let emulatorEnabled: string | undefined;
+
+  beforeEach(() => (emulatorEnabled = process.env.SPANNER_EMULATOR_HOST));
```

Comment:
This check should be able to be set outside of a `before`/`beforeEach` block. And just to be a little more accurate with the var name's meaning, can we convert this value to a boolean?:

```ts
describe('Spanner', () => {
  const IS_EMULATOR_ENABLED = typeof process.env.SPANNER_EMULATOR_HOST !== 'undefined';
```

Reply:
Done.